code/game/gamemodes/gang/gang.dm

```diff
@@ -0,0 +1,449 @@
+			fuckingdone = TRUE
+			send_in_the_fuzz()
+		check_counter = 0
+		SSticker.mode.check_win()
+
+		check_tagged_turfs()
+		check_gang_clothes()
+		check_rollin_with_crews()
+
+/datum/game_mode/gang/proc/send_in_the_fuzz()
+	var/team_size
+	var/cops_to_send
+	var/announcement_message = "PUNK ASS BALLA BITCH"
+	var/announcer = "Spinward Stellar Coalition"
+	if(GLOB.joined_player_list.len > LOWPOP_FAMILIES_COUNT)
+		switch(GLOB.deaths_during_shift)
+			if(0 to TWO_STARS_HIGHPOP-1)
+				team_size = 8
+				cops_to_send = /datum/antagonist/ert/families/beatcop
+				announcement_message = "Hello, crewmembers of [station_name()]! We've received a few calls about some potential violent gang activity on board your station, so we're sending some beat cops to check things out. Nothing extreme, just a courtesy call. However, while they check things out for about 10 minutes, we're going to have to ask that you keep your escape shuttle parked.\n\nHave a pleasant day!"
+				announcer = "Spinward Stellar Coalition Police Department"
+			if(TWO_STARS_HIGHPOP to THREE_STARS_HIGHPOP-1)
+				team_size = 9
+				cops_to_send = /datum/antagonist/ert/families/beatcop/armored
+				announcement_message = "Crewmembers of [station_name()]. We have received confirmed reports of violent gang activity from your station. We are dispatching some armed officers to help keep the peace and investigate matters. Do not get in their way, and comply with any and all requests from them. We have blockaded the local warp gate, and your shuttle cannot depart for another 10 minutes.\n\nHave a secure day."
+				announcer = "Spinward Stellar Coalition Police Department"
+			if(THREE_STARS_HIGHPOP to FOUR_STARS_HIGHPOP-1)
+				team_size = 10
+				cops_to_send = /datum/antagonist/ert/families/beatcop/swat
+				announcement_message = "Crewmembers of [station_name()]. We have received confirmed reports of extreme gang activity from your station resulting in heavy civilian casualties. The Spinward Stellar Coalition does not tolerate abuse towards our citizens, and we will be responding in force to keep the peace and reduce civilian casualties. We have your station surrounded, and all gangsters must drop their weapons and surrender peacefully.\n\nHave a secure day."
+				announcer = "Spinward Stellar Coalition Police Department"
+			if(FOUR_STARS_HIGHPOP to FIVE_STARS_HIGHPOP-1)
+				team_size = 11
+				cops_to_send = /datum/antagonist/ert/families/beatcop/fbi
+				announcement_message = "We are dispatching our top agents to [station_name()] at the request of the Spinward Stellar Coalition government due to an extreme terrorist level threat against this Nanotrasen owned station. All gangsters must surrender IMMEDIATELY. Failure to comply can and will result in death. We have blockaded your warp gates and will not allow any escape until the situation is resolved within our standard response time of 10 minutes.\n\nSurrender now or face the consequences of your actions."
```

Comment:
@Iamgoofball shouldn't this have been "Nanotrasen-owned" instead of "Nanotrasen owned"?